easybuild/easyconfigs/c/CMake/CMake-3.9.4-foss-2017a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.9.4'
+
+homepage = 'http://www.cmake.org'
+
+description = """
+ CMake, the cross-platform, open-source build system.  CMake is a family of
+ tools designed to build, test and package software.
+"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b5d86f12ae0072db520fdbdad67405f799eb728b610ed66043c20a92b4906ca1']
+
+builddependencies = [
+    ('binutils', '2.27'),
```

Comment:
this makes no sense, a full toolchain like `foss` already includes `binutils`...